### ISSUE TITLE
donate button

### ISSUE THREAD

**Issue by CitroenGames** [InoriRus/Kyty#35]
hi,
i know poeple would like to donate to support the project so i think you should add a donate button.

**Comment by LifeIsStrange** [comment]
https://github.com/sponsors

**Comment by Aghneel** [comment]
I would like to donate too

**Comment by Mored4u** [comment]
> hi, i know poeple would like to donate to support the project so i think you should add a donate button.

I think it will push the project to go extremely fast

**Comment by ultimaweapon** [comment]
I don't think so, unless the amount is large enough for the developer to work in a full-time. Even if the amount if large enough it is still not guarantee the developer will leave their job to work on this as a full-time.

IMO if we really want this project to go fast we should help on what we can do.

**Comment by bruhgaming12** [comment]
But even then it would motivate the dev maybe.
